docs/source/data/subscriptions.mdx

```diff
@@ -252,6 +246,7 @@ npm install graphql-subscriptions
 ```
 
 A `PubSub` instance enables your server code to both `publish` events to a particular label and listen for events associated with a particular label. We can create a `PubSub` instance like so:
```

Comment:
(scroll up a few lines)

> Apollo Server uses a **publish-subscribe** (**pub/sub**) model to track events that update active subscriptions

we should probably remove all things that claim that *Apollo Server* does anything relating to subscriptions (should have been done w/ v3)

Reply:
Great point! Tweaked this to:

> You can use the **publish-subscribe** (**pub/sub**) model to track events that update active subscriptions.